.env.example

```diff
@@ -1,4 +1,5 @@
 # Wallarm API settings
+NODE_IMAGE = "wallarm/node:test"
```

Comment:
is this variable required for local build and test? 

`NODE_MAGE` overwrites here by `$(IMAGE)`
https://github.com/wallarm/docker-wallarm-node/pull/101/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R25

Reply:
no, that parameter is not required for local start.